download/nightly/index.md

```diff
@@ -128,16 +165,12 @@ a certain build is not available for your operating system, please check the pre
         const summary = document.createElement("summary");
         if ("commit" in data) {
           // Build summary - timestamp + (linked) commit message
-          summary.appendChild(document.createTextNode(`${data.commit.timestamp.replace("T", "_").replace("Z", "")} \u2013 `));
+          const timestamp = new Date(data.commit.timestamp);
+          const dateMessage = document.createTextNode(timestamp.toLocaleString(undefined, {"dateStyle": "medium"}) + " \u2013 ");
```

Comment:
The average user is going to want to know things like "when was the most recent update" or "how frequent are updates"? They are unlikely to check for updates from the same day. The raw date/time feels very unfriendly for users. If a more advanced user wants to compare versions that's fine, but it is best if they do it with the commit checksum anyway, not the date and time. Imagine for instance that a build was manually restarted. Then there could potentially be two separate dates with the same build.

Reply:
Actually no, because the date is taken from the commit information. But it doesn’t matter. You are right, the shorter format is more appropriate. But I still think we should use the ISO date format here, not some local format. German dates on an English site are confusing. The ISO format also avoids that entire European vs. American date mess.

Reply:
I feel like there's still some opportunity for confusion with the ISO date format that is unavoidable with any all-numeric date format, which is why I went for the slightly long abbreviated full month. How about fixing the locale to "en-us" rather than using the user's locale for the date formatting?

Reply:
> I went for the slightly long abbreviated full month (although I don't know if "medium" does that in all locales)

It certainly doesn’t do that in the German locale. But if en-us includes the month as a well then we can use that too, sure.

Reply:
Give it another try now. I just looked into it a bit and `toLocaleString` can actually can vary between browsers, so it might not be the ideal thing to use, but unfortunately the other option seems to be adding an entire library which seems like overkill to me.

Reply:
Yeah I agree with that. But I tried Firefox, Chromium and surf (a WebKit browser) and they all seemed to include the abbreviated month, so I suppose it should be fine.

I’ll take another look at the PR as a whole to make sure we didn’t miss anything and then merge it.